src/modules/commander/state_machine_helper.cpp

```diff
@@ -1194,13 +1195,9 @@ void battery_failsafe(orb_advert_t *mavlink_log_pub, const vehicle_status_s &sta
 
 		// FALLTHROUGH
 		case LOW_BAT_ACTION::LAND:
-			if (TRANSITION_DENIED != main_state_transition(status, commander_state_s::MAIN_STATE_AUTO_LAND, status_flags,
```

Comment:
@julianoes Sorry, I can't comment on a line which you did not change but don't you need to put the LOW_BAT_ACTION::RETURN_OR_LAND case before the LOW_BAT_ACTION::RETURN case, as you have done for the critical battery level section? As I understand it will otherwise fall through to landing and never RTL.

Reply:
I think this is correct because in the emergency case we want to land, so we fallthrough to land.